.github/workflows/evergreen.yaml

```diff
@@ -40,6 +40,15 @@ jobs:
       platform: evergreen-arm-hardfp
       nightly: ${{ github.event.inputs.nightly }}
       run_api_leak_detector: true
+  evergreen-arm-hardfp-rdk:
+    uses: ./.github/workflows/main.yaml
+    permissions:
+      packages: write
+      pull-requests: write
+    with:
+      platform: evergreen-arm-hardfp
```

Comment:
`evergreen-arm-hardfp` here should say `[evergreen-arm-hardfp-rdk`

Reply:
updated.